src/controller/LottoController.js

```diff
@@ -0,0 +1,105 @@
+import LottoGenerator from '../domains/LottoGenerator';
+import LottoCalculator from '../domains/LottoCalculator';
+import InputView from '../views/InputView';
+import OutputView from '../views/OutputView';
+import LottoPaymentValidator from '../validators/LottoPaymentValidator';
+import LottoValidator from '../validators/LottoValidator';
+import executeWithRetry from '../utils/executeWithRetry';
+import LOTTO_RULES from '../constants/lotto-rules';
+
+class LottoController {
+  #lottoNumbers = {};
+
+  constructor() {
+    this.#lottoNumbers = {
+      winningNumbers: null,
+      bonusNumber: null,
+    };
+  }
+
+  async run() {
+    const tickets = await this.initTicketCount();
+    const lottoGenerator = await this.initLottoGenerator(tickets);
+    await this.initLottoNumbers();
+    await this.initLottoCalculator(tickets, lottoGenerator);
+    await this.reStartLotto();
+  }
+
+  async initTicketCount() {
+    const tickets = await this.readLottoPayment();
+    return this.getTicketCount(tickets);
+  }
+
+  async initLottoGenerator(tickets) {
+    const lottoGenerator = new LottoGenerator(tickets);
+    OutputView.printLottoPayment(tickets);
+    OutputView.printGeneratedLottos(lottoGenerator.generatedLottos);
+    OutputView.printNewLine();
+    return lottoGenerator;
```

Comment:
가독성을 위해 선언부 / 중간 실행부 / 반환문 사이에 개행을 넣어주는 것도 좋아요.

```suggestion
    const lottoGenerator = new LottoGenerator(tickets);
    
    OutputView.printLottoPayment(tickets);
    OutputView.printGeneratedLottos(lottoGenerator.generatedLottos);
    OutputView.printNewLine();
    
    return lottoGenerator;
```